CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## How to contribute
+We love your contributions. Take a look at these guidelines to know
+how to contribute to ubuntu-report.
+
+## Getting started
+Make sure you have the repo cloned under `$GOPATH/src` by using the `go get ... ` command.
+You can learn more about this [here](https://golang.org/cmd/go/#hdr-Download_and_install_packages_and_dependencies)
+
+## Formatting using gofmt
```

Comment:
I wouldn't mention the befinits or con using go fmt. It's standard in the Go community to use it, and ship with the default tools. Maybe remove this paragraph and only let that this project follow go fmt as standard go style. (You can keep the link, but rephase the "click here") please

Reply:
What about
"This project uses `gofmt` as standard code style. You can learn more about gofmt [here](https://blog.golang.org/go-fmt-your-code)."